app/views/admin/splits/_details.html.erb

```diff
@@ -33,5 +35,11 @@
         <td>&nbsp;</td>
       </tr>
     </table>
+    <% if @split.feature_gate? %>
+      <p>
+      <a id="gate_population"></a>
+      * Feature gates do not track assignment events and population reflects only visitors assigned to specific variants.
```

Comment:
👍 can we make this even more explicit, maybe `only visitors assigned to specific variants via the chrome extension or admin tool` or something to make clear that if 10 Betterment employees have it on it will show population 10 even if the weight is 20% `true` and thousands fo people have seen the `true` experience

Reply:
done.